ivy/functional/frontends/tensorflow/signal.py

```diff
@@ -0,0 +1,7 @@
+import ivy
+from ivy.functional.frontends.tensorflow.func_wrapper import to_ivy_arrays_and_back
+
+
+@to_ivy_arrays_and_back
+def dct(input, type=2, n=None, axis=-1, norm=None, name=None):
+    return ivy.dct(input, type, n, axis, norm, name)
```

Comment:
I do not think there is an ivy.dct function for you to use directly, please check the deep dive document https://lets-unify.ai/ivy/deep_dive/ivy_frontends.html#compositions for what to do in this case. Thank you.

Reply:
Hi Daniel, have created a new issue according to the documentation, but i was unable to add the labels as stated on the documentation, any way forward?

Reply:
In this case, I will add the labels for you. Can you give me the link to the issue? Thank you.